planner/core/mock.go

```diff
@@ -263,6 +263,51 @@ func MockTable() *model.TableInfo {
 	return table
 }
 
+// MockTableT2 is only used for plan related tests.
+func MockTableT2() *model.TableInfo {
```

Comment:
when should we use `MockTableT` and `MockTableT2`?

Reply:
One holds a signed pk and one holds a unsigned pk. Maybe there'll be a table hold not integer pk in the future.

Reply:
Can we put this information in the name? Like `MockSignedPKTable`, we could tell the difference without looking into the code.

Reply:
Addressed.